plugin/src/main/resources/io/jenkins/plugins/forensics/git/reference/GitReferenceRecorder/config.jelly

```diff
@@ -4,6 +4,11 @@
   <f:entry title="${%title.referenceJob}" field="referenceJob" description="${%description.referenceJob}">
     <f:combobox/>
   </f:entry>
+  
+  <f:entry title="${%title.referenceBuildFinderStrategy}" description="${%description.referenceBuildFinderStrategy}"
+           field="referenceBuildFinderStrategy">
+    <f:select default="PARENT_COMMIT_BUILD"/>
```

Comment:
It depends somewhat on the implementation but I think the default should be the other way round.